node_modules/oae-rest/README.md

```diff
@@ -0,0 +1,21 @@
+The `rest` module for OAE.
```

Comment:
I updated gitignore to ignore the oae-rest module and I added a pull request for the oae-rest project : https://github.com/oaeproject/oae-rest/pull/29.